src/test/functional/functional-tests.js

```diff
@@ -14,20 +14,21 @@
  * limitations under the License.
  */
 
-import { Policy } from '../../../dist/main/minio'
-import * as Minio from '../../../dist/main/minio'
-import os from 'os'
-import stream from 'stream'
-import crypto from 'crypto'
-import async from 'async'
-import _ from 'lodash'
-import fs from 'fs'
-import http from 'http'
-import https from 'https'
-import url from 'url'
-import superagent from 'superagent'
-import { assert } from 'chai'
-import uuid from 'uuid'
+const minio = require('minio');
```

Comment:
This fails the CI because this isn't a valid import when the package isn't installed. Perhaps we could add a try / catch, that first tries `require('minio')`, then reverts back to the path in `dist`? This would allow both to happen.

Also, if you change this to `const Minio = require('minio');` you can get rid of the caps changes below.